core/orchestrator.go

```diff
@@ -98,30 +98,48 @@ func (orch *orchestrator) TranscoderResults(tcId int64, res *RemoteTranscoderRes
 }
 
 func (orch *orchestrator) ProcessPayment(payment net.Payment, manifestID ManifestID) error {
+
 	if orch.node == nil || orch.node.Recipient == nil {
 		return nil
```

Comment:
@yondonfu Should we return an error here instead of nil or not (analogous to the two checks below) ?

Reply:
We should return `nil` here because the orchestrator could be running in off-chain mode (i.e. broadcaster is directly connecting to an orchestrator without any interactions with the blockchain). `orch.node.Recipient = nil` when the orchestrator is running in off-chain mode and we wouldn't want to return an error in this scenario - returning nil allows the orchestrator to proceed with the transcoding by ignoring the PM related logic.